wp-content/themes/workingnyc/src/js/modules/programs-archive.js

```diff
@@ -140,6 +158,33 @@ export default {
     };
   },
 
+  computed: {
```

Comment:
```suggestion
  computed: {
  // The two functions below override functions in archive.vue because this implementation 
  // has only one page's data stored at a time, while the archive.vue implementation stores data
  // on multiple pages
```